README.md

```diff
@@ -1,41 +0,0 @@
-# silverstripe-freshdesk
-
-SilverStripe module which allows CMS admins to configure userforms to raise tickets with freshdesk.
```

Comment:
It might make sense to split SSO and ticketing components. These two don't seem coupled in any way.